RELEASE.md

```diff
@@ -16,6 +16,11 @@
 ## v0.1.10 - 04/13/2023
 - Added OC2 Realm.
 
+## v1.1.10 - 01/19/2023
```

Comment:
May need to double check the release version

Reply:
Should be v1.1.0

Reply:
and it's 01/19/2024

Reply:
updated!